src/Csv.cpp

```diff
@@ -126,10 +143,35 @@ QString Csv::writeSelectionToString(QAbstractItemModel *model, bool includeHeade
     return text;
 }
 
-QString Csv::escape(QString value)
+QString Csv::escape(QString value, bool quoteStringData)
 {
     //TODO replace this with a real csv library
     if (value.contains(m_delimiter) || value.contains("\n") || value.contains(m_quote))
         return m_quote + value.replace(m_quote, m_quote + m_quote) + m_quote;
-    return value;
+    else if (quoteStringData)
```

Comment:
The name "quoteStringData" had me confused for a bit, I take it this is an option to always quote string values, even when there is no delimiter present in the string (minimal quote vs always quote)? Might consider a name change as it didn't highlight the different between m_quote (noun) and quoteStringData (verb) immediately.

Reply:
ok. i'll merge it then later fix it.